src/main/java/duke/tasks/Todo.java

```diff
@@ -0,0 +1,33 @@
+package duke.tasks;
+
+/**
+ * Encapsulates a Todo type task
+ */
+public class Todo extends Task {
+    private TaskType type = TaskType.TODO;
+    public Todo (String description){
```

Comment:
Try to have consistent formatting 😄 
```suggestion
    public Todo (String description) {
```